content/en/metrics/otlp.md

```diff
@@ -0,0 +1,218 @@
+
+---
+title: OTLP Metrics Types
+kind: documentation
+aliases:
+    - /developers/metrics/otlp/
+further_reading:
+    - link: 'metrics/distributions'
+      tag: 'Documentation'
+      text: 'Learn more about distributions'
+    - link: 'tracing/setup_overview/open_standards/'
+      tag: 'Documentation'
+      text: 'Learn more about OpenTelemetry'
+---
+
+## Overview
+
+The Datadog Agent and the OpenTelemetry Collector Datadog exporter can ingest metrics in the OpenTelemetry format (OTLP), which can be produced by OpenTelemetry-instrumented applications.
+
+The following OTLP metric types can be ingested by the Datadog Agent and OpenTelemetry Collector Datadog exporter:
+- Sums,
+- Gauges,
+- Histograms and
+- Summaries.
+
+These different OTLP metric types are mapped to Datadog in-app metric types found within the Datadog web application:
+
+- COUNT
+- GAUGE
+- DISTRIBUTION
+
+A single OTLP metric may be mapped to several Datadog metrics with a suffix indicating their meaning.
+
+**Note**: OTLP metric types are commonly produced by but different from OpenTelemetry metrics API instruments. The OpenTelemetry SDK allows for customization of the OTLP metrics produced by a given instrument.
```

Comment:
The point of the note is to briefly say that we are talking about OTLP types and not other OpenTelemetry types. Let me explain in more detail so we can think about how to rephrase this. 

The most common data pipeline in OpenTelemetry, from user code to Datadog has four steps:
OpenTelemetry API Instrument  → OpenTelemetry SDK → OTLP payload → Datadog

The note says two things:

1. > OTLP metric types are commonly produced by but different from OpenTelemetry metrics API instruments.
   
    This states that OTLP types are different to OpenTelemetry API instruments. Some OTLP types have similar names to the API instruments, e.g. there is an OpenTelemetry API Histogram and an OTLP Histogram, so it's important to say that we are talking about OTLP types to avoid confusion.

2. > The OpenTelemetry SDK allows for customization of the OTLP metrics produced by a given instrument.

    This states that the same OpenTelemetry API instrument may produce different OTLP types depending on the SDK configuration. For example, an OpenTelemetry API Counter instrument may produce an OTLP Cumulative Sum or an OTLP Delta Sum depending on SDK configuration.

Reply:
I think being a bit more verbose (like what you posted, with examples) would be nice, since that note aims at making sure customers understand the difference between what the instrument types vs. SDK types are.

Like 

"The OpenTelemetry metrics API defines multiple instruments (`Counter`, `UpDownCounter`, etc.), that are translated into OTLP metric types (`Sum`, `Gauge`, etc.). The OpenTelemetry SDK allows for customization  of the OTLP metric types produced by a given instrument (eg. depending on configuration, an `UpDownCounter` could be translated into a `Sum` or a `Gauge`).

In the sections below, the "metric types" refer to the OTLP metric types, not the metrics API instruments."

(there's probably a better wording for this).

Reply:
Thanks! I will try to make this a bit more verbose. It's hard to keep the balance between avoiding confusion and not re-explaining what is/will be already available in the OpenTelemetry docs, but at least adding an example or two makes sense, since this will be the most common way of getting OTLP payloads

Reply:
I have reworded this a bit; after thinking about this more I think adding concrete examples now may be confusing, since most language libraries are still not finished with their implementation of the metrics SDK spec and their behavior varies wildly.

Once the SDK is ready, we can add examples below with the instruments. PTAL!